### ISSUE TITLE
hyperopt is a really bad name

### ISSUE THREAD

**Issue by berndbischl** [jakob-r/mlrHyperopt#3]
because of this
https://github.com/hyperopt/hyperopt

**Comment by jakob-r** [comment]
Yes and no. Yes because I obviously don't want to generate the illusion that mlrHyperopt is a link between mlr and the TPE part of hyperopt. No because hyperopt seems to be the obvious short version for hyperparameter optimization and hyperopt itself is not a package entirely devoted to one specific optimization method and it lives in the Python world.

**Comment by Prometheus77** [comment]
I was working on a much less sophisticated optimization wrapper before I discovered mlrMBO, basically an iterative grid search that iteratively zooms in on better performing parameter spaces. I was planning on calling it "autotune", both because the name makes sense for the problem we're trying to solve, and because it's catchy and memorable ... it's what they call this technique in pop music recording: https://youtu.be/koQksuxzJ4w

Just a thought!
